### PR TITLE
secretsutil cleanup

### PR DESCRIPTION
**Pull request by wackxu**
Signed-off-by: x00425595 <xushiwei5@huawei.com>

**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: cloud/pkg/cloudhub/servers/httpserver/secretsutil.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ func GetSecret(secretName string, ns string) (*corev1.Secret, error) {
 func CreateSecret(secret *corev1.Secret, ns string) error {
 	cli := client.GetKubeClient()
 	if err := CreateNamespaceIfNeeded(cli, ns); err != nil {
-		return fmt.Errorf("failed to create Namespace kubeedge, error: %s", err)
+		return fmt.Errorf("failed to create Namespace kubeedge, error: %v", err)
```
Comment (zirain) [inline cloud/pkg/cloudhub/servers/httpserver/secretsutil.go:36]: `%w` is better?

Comment (wackxu) [reply cloud/pkg/cloudhub/servers/httpserver/secretsutil.go:36]: from  https://github.com/golang/go/wiki/ErrorValueFAQ#i-am-already-using-fmterrorf-with-v-or-s-to-provide-context-for-an-error-when-should-i-switch-to-w

I think use %v is better for now

Comment (gy95) [reply cloud/pkg/cloudhub/servers/httpserver/secretsutil.go:36]: LGTM

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**